site/content/entry/yuzu-progress-report-dec-2021/index.md

```diff
@@ -0,0 +1,376 @@
+[emulation was paused and the window was out of focus](https://github.com/yuzu-emu/yuzu/pull/7506). 
+Thank you!
+
+[jbeich](https://github.com/jbeich) changed the building configuration so that 
+[VA-API, one of the video decoding APIs of Linux, is enabled on Unix systems](https://github.com/yuzu-emu/yuzu/pull/7602), allowing the users who want to build targeting 
+BSD or other Unix-based systems to use hardware acceleration for video decoding.
+
+This is just one of several PRs jbeich wrote to help yuzu work on BSD systems, thank you for your contributions!
+
+## UI changes
+
+The favourites row in yuzu’s game list was always expanded, even if the user collapsed it.
+epicboy [added a persistent setting](https://github.com/yuzu-emu/yuzu/pull/7570) to remember the user preference between launches.
+
+One of the most common issues users face is lack of Vulkan support on their PC. Not lack of hardware support, but instead missing software support caused by outdated GPU 
+drivers or poorly coded/outdated Vulkan injections.
+
+Our old error popup didn’t reflect this so [your writer](https://github.com/goldenx86), with his total lack of coding skills, 
+[decided to change it](https://github.com/yuzu-emu/yuzu/pull/7532).
+
+{{< single-title-imgs
+    " "
+    "./vidbug.png"
+    "./vidfix.png"
+    >}}
+
+This is a complex issue and the main reason Vulkan is not the default API in use.
+Users of old laptops with AMD and Intel integrated GPUs tend to use the driver shipped by either the laptop vendor or Windows Update. 
+In both cases those drivers are most likely years old (yuzu can run on AMD GPUs from 2012) and either lack Vulkan support at all, or only support a portion of what’s 
+needed to run yuzu.
+Also, since laptops by default connect the display directly to the integrated GPU, that’s the first Vulkan driver that will be seen, so it’s critical to have the latest 
+GPU driver installed *even if* your laptop has a dedicated NVIDIA GPU running the latest driver.
+
+While telling AMD users to [manually download and install updated drivers](https://www.amd.com/en/support) is a viable option and works as it should, in its 
+*infinite wisdom*, Intel decided to block manual installation of its own official drivers if a custom laptop vendor driver is in use (those modified drivers are usually 
+created to cheat on battery life metrics and/or to save money on cooling).
+
+The only alternative in those cases is to [manually download the ZIP version](https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-dch-drivers.html) 
+of the driver, unpack it, go to Device Manager, right-click the correct GPU in Display Adapters, select `Update Driver Software…`, `Browse my computer for driver software`, 
+`Let me pick from a list of device drivers on my computer`, `Have Disk…`, then finally browse to the folder where the driver was unpacked and select the `iigd_dch.inf` file.
```

Comment:
Maybe use a faq article for this and link it or add image

Reply:
Let's add a video to help.